index.html

```diff
@@ -8,135 +8,172 @@
+          <summary class="app-grid__card__show-answer--toggle-show">
             Show Answer
           </summary>
-          <p class="app-grid__main__card__show-answer__answer">8</p>
+          <p class="app-grid__card__show-answer__answer">8</p>
 
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
         </details>
       </section>
 
-      <section class="app-grid__main__card">
+      <section class="app-grid__card">
+        <img
+          src="img/icons/bookmark-regular.svg"
+          alt="bookmark"
+          class="app-grid__card--icon"
+        />
         <h2>5+5?</h2>
-        <details class="app-grid__main__card__show-answer">
-          <summary class="app-grid__main__card__show-answer--toggle-show">
+        <details class="app-grid__card__show-answer">
+          <summary class="app-grid__card__show-answer--toggle-show">
             Show Answer
           </summary>
-          <p class="app-grid__main__card__show-answer__answer">10</p>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <p class="app-grid__card__show-answer__answer">10</p>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <ul class="app-grid__card__tags">
+            <li class="app-grid__card__tags--tag">category</li>
           </ul>
         </details>
       </section>
     </main>
 
-    <footer class="app-grid__footer">
-      <nav class="app-grid__footer__nav">
+    <footer class="footer">
+      <nav class="footer__nav">
         <a> home </a>
-        <a> bookmarks </a>
+        <div class="footer__nav__entry">
+          <img
+            src="img/icons/bookmark-regular.svg"
+            alt="bookmark"
+            class="footer__nav__entry__icon app-grid__footer__nav__entry__icon--bookmark"
```

Comment:
Here you have a lot of classes. Do you need all of them? One class would be enough, I guess.
```suggestion
            class="navigation__icon"
```